lib/extensions.js

```diff
@@ -375,10 +375,14 @@ function getPlatformVariant() {
   if (process.platform !== 'linux') {
     return '';
   }
-  if (fs.readFileSync(process.execPath).indexOf('libc.musl-x86_64.so.1') !== -1) {
-    return 'musl';
-  }
-  return '';
+  var contents = '';
+  try {
+    contents = fs.readFileSync('foo.bar').toString();
+    if (contents.indexOf('libc.musl-x86_64.so.1') !== -1) {
```

Comment:
sorry I sent comment while you are updating this.
toString is slow on where Buffer.indexOf is available. so this would be longer version.

```
try {
  var muslName = 'libc.musl-x86_64.so.1';
  var contents = fs.readFileSync(process.execPath);
  var hasMusl = contents.indexOf ? contents.indexOf(muslName) > 0 : contents.toString().indexOf(muslName) > 0;
  if (hasMusl) {
    return 'musl';
  }
} catch (err) { console.log (err); }
  return '';
}
```